media/css/contentful/mozilla/c-split.scss

```diff
@@ -10,3 +10,13 @@ $brand-theme: 'mozilla';
 @import '~@mozilla-protocol/core/protocol/css/components/split';
 
 @import '../patch';
+
+.mzp-c-split-body.c-home-body {
```

Comment:
After conversations with Craig and Dan, I think its _okay_ that the split is wider than the hero. I think this is a good call out though! And I think if we use a lot of these as hero's it might be a good idea to have a separate content model.